inst/rmarkdown/templates/xaringan/resources/su-fonts.css

```diff
@@ -0,0 +1,13 @@
+@import url(https://fonts.googleapis.com/css?family=Open+Sans);
+@import url(https://fonts.googleapis.com/css?family=Yanone+Kaffeesatz);
+@import url('https://fonts.googleapis.com/css?family=Inconsolata:400,700');
+@import url('https://fonts.googleapis.com/css?family=Actor');
+
+body { font-family: 'Open Sans', 'Palatino Linotype', 'Book Antiqua', Palatino, 'Microsoft YaHei', 'Songti SC', serif; }
```

Comment:
One font for body and monospace is sufficient because they are imported on the fly.
If not available, system fallbacks will be used but I think none of the fonts listed here is installed by default on a system?